{"name": "输入已存在的用户名注册失败", "status": "passed", "attachments": [{"name": "stdout", "source": "661b23b0-1fea-4138-ad3f-9b3cf533a13a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '9', 'casename': '输入已存在的用户名注册失败', 'mode': '注册', 'data': 'userAccount=979172251@qq.com', 'assert_way': 'xpath=/html/body/div[2]/span/div/div/div/span', 'result': 'The mailbox is already registered'}"}], "start": 1651193987341, "stop": 1651193991146, "uuid": "d510bb40-7655-4218-8838-d9aaac4e2a7a", "historyId": "f44382204f6c23bd5640584e5a08c4dd", "testCaseId": "91efbca2877ad29cfc72c696a08d59d8", "fullName": "test_main.Test_Main#test_regist", "labels": [{"name": "feature", "value": "注册"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "23236-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}